source/site-administrators/studio/rte2-setup.rst

```diff
@@ -73,6 +73,40 @@ The default editor instance contains a menubar with most of the commonly used ed
 
 |
 
+.. _rte-paste-plugin-hooks:
+
+TinyMCE paste plugin callback hooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The TinyMCE paste plugin enables you to modify the pasted content before it gets inserted into the editor (``paste_preprocess``) and before it gets inserted into the editor but after it’s been parsed into a DOM structure (``paste_postprocess``).  For more information on these options, see https://www.tiny.cloud/docs/plugins/opensource/paste#paste_preprocess.
+
+In order to hook into the callback (``paste_preprocess`` and ``paste_postprocess``), do the following in the RTE configuration:
+
+1) Add the default ``paste`` plugin in the ``<plugins />`` tag
+
+   .. code-block:: xml
+      :caption: *CRAFTER_HOME/data/repos/sites/SITENAME/sandbox/config/studio/form-control-config/rte/rte-setup-tinymce5.xml*
+
+      <plugins>
+        print preview searchreplace autolink directionality visualblocks visualchars fullscreen image link media template
+        codesample table charmap hr pagebreak nonbreaking anchor toc insertdatetime advlist lists wordcount
+        textpattern help acecode paste
```

Comment:
When Tiny's paste plugin is included, `craftercms_paste_cleanup` extension is also enabled. CrafterCMS' extension performs some additional paste cleanup from what Tiny's plugin does. To disable these additional processing of the paste input, you may add `<craftercms_paste_cleanup>false</craftercms_paste_cleanup>` to the RTE configuration